tests/NuGet.Services.AzureSearch.Tests/SearchDocumentBuilderFacts.cs

```diff
@@ -556,11 +557,35 @@ public void UsesIdWhenMissingForSortableTitle(string title)
                     fullVersion: Data.FullVersion,
                     package: package,
                     owners: Data.Owners,
-                    totalDownloadCount: Data.TotalDownloadCount);
+                    totalDownloadCount: Data.TotalDownloadCount,
+                    isExcludedByDefault: false);
 
                 Assert.Equal(Data.PackageId.ToLowerInvariant(), document.SortableTitle);
             }
 
+            [Theory]
+            [InlineData(true)]
+            [InlineData(false)]
+            public void SetsIsExcludedByDefaultPropertyCorrectly(bool shouldBeExcluded)
+            {
+                var package = Data.PackageEntity;
+                package.Title = "title";
```

Comment:
Shouldn't need to capture `Data.PackageEntity` in a local variable or set `Title`